dockstore-webservice/src/main/java/io/dockstore/webservice/core/LambdaEvent.java

```diff
@@ -0,0 +1,175 @@
+package io.dockstore.webservice.core;
+
+import java.sql.Timestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+/**
+ * This describes events triggered by GitHub webhooks
+ */
+@ApiModel("LambdaEvent")
+@Entity
+@Table(name = "LambdaEvent")
+@NamedQueries({
+        @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByRepository", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.repository = :repository"),
+        @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByUsername", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.username = :username"),
+        @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByUser", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.user = :user"),
+})
+@SuppressWarnings("checkstyle:magicnumber")
+public class LambdaEvent {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @ApiModelProperty(value = "Unique ID of the event.", position = 0)
+    private long id;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The repository from the event.", required = true, position = 1)
+    private String repository;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The name of the user on GitHub that triggers the event.", required = true, position = 2)
+    private String username;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The git reference from the event.", required = true, position = 3)
+    private String reference;
+
+    @Column(nullable = false, columnDefinition = "boolean default true")
+    @ApiModelProperty(value = "Whether or not the event was successful.", position = 4)
+    private boolean success = true;
+
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The message associated with the event.", position = 5)
+    private String message;
+
+    @Column(nullable = false, columnDefinition = "boolean default false")
+    @ApiModelProperty(value = "Whether or not the user has dismissed the event.", position = 6)
```

Comment:
Based on a discussion with Denis, this will only track the user who triggered the event. For other users to be able to see the events, we will use an endpoint which if a user has access, will show all events for a GitHub organisation. This way we don't need to keep track of all the users.

Reply:
Following on, if all users in the org can see the event, it seems that we won't be able to track it with one boolean (I'm ok with not tracking this)

Reply:
Depending on how intrusive the UI will be, I think its fair not to dismiss the event or track which users have seen it. The error should disappear when a new change is pushed (and hopefully fixed). We don't track or dismiss the validation error messages on the files tab for example.